README.md

```diff
@@ -2,3 +2,31 @@
 
 [![CircleCI](https://circleci.com/gh/helloeave/depper.svg?style=svg)](https://circleci.com/gh/helloeave/depper)
 [![GoDoc](https://godoc.org/github.com/helloeave/depper?status.svg)](https://godoc.org/github.com/helloeave/depper)
+
+Depper is used to express constraints on the go package dependency graph. Like its sibling [lidder](https://github.com/helloeave/lidder), depper works by defining various rules. Here is an example
+
+```
+rules:
+  - name: utilities can use other utilities and config
+    packages: util/.*
+    may_depend:
+      - <.*>
+      - third_parties
+      - config
+      - .*util
```

Comment:
Does this also match packages like `github.com/helloeave/eave-go/futil/something`?

Reply:
Yes, it would